### ISSUE TITLE
Can't use predict_celltype for sce converted from Seurat

### ISSUE THREAD

**Issue by veralow1990** [ddiez/celltype#1]
Hi there!
I'm interested in using celltype for data generated from 10X Genomics pipelines.
I used Seurat to read the 10X data and converted the Seurat object to SCE.
When I tried to use predict_celltype to read the resulted SCE, there came an error saying "Error in UseMethod("predict_celltype") : 
  no applicable method for 'predict_celltype' applied to an object of class "c('dgCMatrix', 'CsparseMatrix', 'dsparseMatrix', 'generalMatrix', 'dCsparseMatrix', 'dMatrix', 'sparseMatrix', 'compMatrix', 'Matrix', 'xMatrix', 'mMatrix', 'Mnumeric', 'replValueSp')" ".
How can I fix this problem?
Thank you so much! Looking forward to your reply.
BW

**Comment by ddiez** [comment]
Yes, I've been hit by this problem before. This is caused because Seurat stores data counts as sparse matrices and some of the functions I use do not support them. A temporary solution is to do:

```
assays(sce) <- lapply(assays(sce), as.matrix)
```

where you have to substitute `sce` by the name of your `SingleCellExperiment` object.

For a more permanent solution I will try to push some fix in the near future.
